packages/generator/src/edmx-to-vdm/common/operation.ts

```diff
@@ -1,42 +1,44 @@
 import { pascalCase } from '@sap-cloud-sdk/util';
 import { ServiceNameFormatter } from '../../service-name-formatter';
-import { VdmFunctionImportBase } from '../../vdm-types';
+import { VdmOperationBase } from '../../vdm-types';
 import { SwaggerPath } from '../../swagger-parser';
 import { functionImportDescription } from '../description-util';
 import { EdmxParameter } from '../../edmx-parser/common';
 import { EdmxFunctionImportV2 } from '../../edmx-parser/v2';
-import { EdmxFunctionImportV4 } from '../../edmx-parser/v4';
-import { getFunctionImportParameters } from './action-function-parameters';
+import { EdmxActionImport, EdmxFunctionImportV4 } from '../../edmx-parser/v4';
+import { getOperationParameters } from './operation-parameter';
 
 /**
  * @internal
  */
-export function transformFunctionImportBase(
-  edmxFunctionImport: EdmxFunctionImportV2 | EdmxFunctionImportV4,
+export function transformOperationBase(
```

Comment:
[pp] Also here a JSdoc to explain that the operations represent funciton and actikns could help.